x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -412,7 +417,35 @@ export const EndpointList = () => {
       }
     >
       {hasSelectedEndpoint && <EndpointDetailsFlyout />}
-      {listData && listData.length > 0 && (
+      {
+        <>
+          <div style={{ display: hasListData ? 'flex' : 'none', maxWidth: 200 }}>
+            <EuiSuperDatePicker
+              onTimeChange={() => {}}
```

Comment:
same here. Or in this case, you could even create a top-level module const (like `const NOOP = () => {}`)

There are a few other callbacks below that should be defined above with `useCallback()` and then that value used here in order to not trigger unnecessary re-renders.

Reply:
will make the changes